src/test/integration/IntegrationTest.kt

```diff
@@ -70,26 +98,75 @@ class IntegrationTest {
     }
 
     /**
-     * Query Iroha account balance
+     * Sends transaction to Iroha.
+     * @return hex representation of transaction
      */
-    fun queryIroha() {
-        IrohaInitializtion.loadIrohaLibrary()
-            .failure {
-                println(it)
-                System.exit(1)
-            }
+    fun sendTxToIroha(txBuilder: ModelTransactionBuilder): String {
+        val utx = txBuilder.build()
+        val hash = utx.hash().hex()
+
+        // sign transaction and get its binary representation (Blob)
+        val txblob = ModelProtoTransaction(utx).signAndAddSignature(keypair).finish().blob().toByteArray()
+
+        // create proto object
+        var protoTx: BlockOuterClass.Transaction? = null
```

Comment:
I would suggest to add some util/ package to minimise code needed to deal with iroha API (in future)

Reply:
Yes, but now we have more prioritised tasks to do.